Makefile

```diff
@@ -20,21 +20,21 @@ chain-spec-builder:
 polkatool:
 	cargo install --path vendor/polkavm/tools/polkatool
 
-qf-run: qf-node
+qf-run: qf-release
 	output/qf-node --dev --tmp --rpc-cors all
 
-qf-run-wasm: qf-node
+qf-run-wasm: qf-release
 	output/qf-node --dev --tmp --rpc-cors all --wasm-runtime-overrides output
 
-qf-node-release: qf-runtime
```

Comment:
`qf-node-release` mentioned in `README.md` but does not exist in the updated Makefile. Please check for the similar issue with other commands expected and in other files like workflow config for instance.